pkg/operator/controllers/workaround/ifreload.go

```diff
@@ -0,0 +1,54 @@
+package workaround
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"github.com/sirupsen/logrus"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/client-go/kubernetes"
+
+	"github.com/Azure/ARO-RP/pkg/util/version"
+)
+
+const (
+	kubeNamespace       = "openshift-azure-ifreload"
+	kubeServiceAccount  = "system:serviceaccount:" + kubeNamespace + ":default"
+	ifreloadImageFormat = "%s.azurecr.io/ifreload:109810fe"
+)
+
+type ifReload struct {
+	log          *logrus.Entry
+	cli          kubernetes.Interface
+	versionFixed *version.Version
+}
+
+func (*ifReload) Name() string {
+	return "ifReload"
+}
+
+func (i *ifReload) IsRequired(clusterVersion *version.Version) bool {
+	return clusterVersion.Lt(i.versionFixed)
```

Comment:
note: this will keep the workaround for older clusters (i think there are still a couple).